netbout-web/src/main/java/com/netbout/rest/RqAlias.java

```diff
@@ -111,7 +112,16 @@ public Alias alias() throws IOException {
                 "/login/start"
             );
         }
-        return aliases.iterate().iterator().next();
+        final Alias alias = aliases.iterate().iterator().next();
+        if (alias.photo().equals(Alias.BLANK)) {
```

Comment:
@dmzaytsev wait... maybe we can do it even better. First, read current `photo()` and then compare it with the avatar from `identity`. If they are not equal, save the new value.